tools/delete_image.bash

```diff
@@ -18,6 +18,10 @@ if [ "$1" = "release" ]; then
     EXECMODE="release"
     shift
 fi
+if [ "$1" = "CD" ]; then
+    EXECMODE="CD"
```

Comment:
Id the execmode CD or release?

Reply:
I'm unsure how i implemented it, but as far as i remember the CD execmode was "release" with an override of the image

Reply:
Yes exactly. The CD execmode overrides the image name and then uses the release exec mode. Problem up until now was that this only happens, if CD is passed explicitely to the script and not if it is the default exec mode. Hence the slight duplication

Reply:
but you are right. for container deletion the execmode should be set to release for the CD image